INSTALL.md

```diff
@@ -125,7 +125,7 @@ In case none of the binary packages listed on our website work for your system,
 The standard GNU/Linux tools and Python are minimal requirements to compile Shogun.
 To compile the interfaces, in addition to [swig](http://www.swig.org/) itself, you will need language specific development packages installed, see [interfaces](#manual-interfaces) below.
 
-There is a larger number of optional requirements.
+There is a larger number of optional requirements.Install dependencies like [ply](https://pypi.python.org/pypi/ply) ,[ctags](https://packagecontrol.io/packages/CTags)  if not found in the output lists of cmake. 
```

Comment:
Nitpicking a bit, but please insert a white space between the . and Install and remove one of the two white spaces between the ) and if.